feature/experimental/p4rt/otg_tests/google_discovery_protocol_packetin_test/google_discovery_protocol_packetin_test.go

```diff
@@ -0,0 +1,529 @@
+// Copyright 2022 Google LLC
```

Comment:
There have been some deeper changes to the ATE version of this file and it will worth it to resync to the new version. testTraffic/configureATE functions from the OTG version should apply cleanly, but testTraffic will need to be changed to return the packet count from the flow.

Reply:
All the changes are ported to the test